anvil.star

```diff
@@ -0,0 +1,57 @@
+ANVIL_IMAGE = "ghcr.io/foundry-rs/foundry:latest"
+ANVIL_PORT = 8545
+ANVIL_BLOCK_TIME = 1
+ANVIL_SLOTS_IN_AN_EPOCH = (
+    2  # Setting to X leads to block N-(X+1) being finalized, being N latest block
+)
+STATE_PATH = "/tmp"
+
+
+def run(plan, args):
+    chain_id = str(args["l1_chain_id"])
+    service_files = {}
+    mnemonic = args.get("l1_preallocated_mnemonic")
+
+    cmd = (
+        "anvil --block-time "
+        + str(ANVIL_BLOCK_TIME)
+        + " --slots-in-an-epoch "
+        + str(ANVIL_SLOTS_IN_AN_EPOCH)
+        + " --chain-id "
+        + chain_id
+        + " --host 0.0.0.0 --port "
+        + str(ANVIL_PORT)
+        + " --dump-state "
+        + STATE_PATH
+        + "/state_out.json"
+        + " --balance 1000000000"
+        + ' --mnemonic "'
+        + mnemonic
+        + '"'
+    )
+
+    load_state = bool(args.get("anvil_state_file"))
```

Comment:
We'll need to figure a way to test this scenario in CI in a follow-up PR imho.

Reply:
Yes, my intention is to make a test using the same example than on the doc file, just delayed it to a new pr :dancers: